010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,119 @@
 ---
 title: What is Xata?
 navTitle: What is Xata?
-keywords: ['severless']
-description: Use integrated cloud services to revolutionize data operations
+keywords: ['severless', 'data platform', 'PostgreSQL', 'architecture']
+description: An overview of Xata, the data platform for PostgreSQL.
 slug: concepts/serverless-data-platform
 published: true
 ---
 
-Xata is a new type of Cloud service: it combines multiple types of stores (relational database, search engine, analytics engine) into a single service. The combined functionality is exposed over a single consistent API. It is also vertically integrated: an advanced admin UI and high-level SDKs come into the package.
+Xata is a serverless data platform for PostgreSQL that optimizes for reliability, scalability, and developer experience.
 
-We call this type of service a _Serverless Data Platform._
+On top of the usual Postgres functionality, Xata provides a set of features that make it easier to build modern applications:
 
-![Serverless Data Platform](images/serverless-data-platform.png)
+- **Branching**: Create a new branch of your data to work on new features or experiments without affecting the main branch. The branching can also be used for tenant separation, allowing you to create a branch for each of your customers.
+- **Zero-downtime schema migrations**: Make changes to your production database with zero-downtime, reversible schema migrations and data backfilling. This functionality is powered by our open-source project [pgroll](https://github.com/xataio/pgroll).
+- **File attachments**: Store and retrieve files up to 1GB, including images, videos, and audio files. The files are stored in object storage and automatically cahced on a global CDN for fast delivery. From the application point of view, however, it's just another column in the database.
+- **Full-text search**: Data is automatically replicated into a full-text search engine, making it easy to build search experiences, supporting fuzzy search, weights, and relevancy boosters.
+- **Vector search**: Easily store embeddings and implement similarity search to power your AI applications.
+- **Admin UI**: A web-based console to manage your data, including schema and data.
```

Comment:
I don't know why, but I always bock when `UI` is used to describe our app. What if we went with the marketing description we landed on for the homepage?

```suggestion
- **Collaborative dashboard**: A web-based console to manage your data, including schema and data.
```